### DISCUSSION TITLE
Support github team reviewers

### DISCUSSION THREAD

**Discussion by tadam313** [lindell/multi-gitter#328] [Ideas]
We use multi-gitter quite regularly to change bulk changes in the github org for all the repositories. Sometimes these changes needs to be reviewed by a team. Even though we can workaround it by listing all the team members as reviewers but it would be more convenient to be able to specify a team reviewer. I believe it comes down to this [code part](https://github.com/lindell/multi-gitter/blob/master/internal/scm/github/github.go#L327) where we'd need to set `TeamReviewers` property rather than just `Reviewers`. 

Perhaps we'd be able to auto detect if some of the member of `reviewers` setting is a team and not a user? Like checking for a format something like `%s/%s` might work, then we don't even need to create another property for team reviewers as I believe it's only GH specific.

If this is something that you consider supporting. WDYT?

**Comment by lindell** [comment]
Seems reasonable to add it 🙂 

I think a new flag/option should be best, even if it's only GH-specific. 

Would you be interested in adding this?

**Reply by tadam313** [reply]
yeah sure I can take a look in coming days! Sure the flag would be the most explicit way, I'll go in that direction then!

**Reply by tadam313** [reply]
@lindell I'm taking a look at this now, and have a question. If we're adding a new flag only for github team reviewers, should we also add a new `--max-team-reviewers` flag to support randomising or should we use the existing `--max-reviewers` flag somehow? 🤔 

I'd probably suggest the first option having another flag for this, but I'm slightly concerned about the number of flags might hurt API clarity. WDYT?

**Reply by lindell** [reply]
Out of the two options, I would agree that no. 1 is the best 👍 

Although I don't find that option necessary at all, if you don't have a need for it. Since it seems to be a very rare usecase, I think we could skip it altogether. If you do have the need for it, feel free to add it with `--max-team-reviewers` 😄

**Comment by lindell** [comment]
For future reference, @tadam313 added this and it was released in [v0.47.0](https://github.com/lindell/multi-gitter/releases/tag/v0.47.0)
